test/integration/crontab_test.exs

```diff
@@ -96,6 +96,36 @@ defmodule Oban.Integration.CrontabTest do
     refute_receive {:ok, 2}
   end
 
+  test "reboot jobs are enqueued on startup" do
```

Comment:
Looking at these tests now I realize that we don't need to execute the job to make assertions at all. Using `queues: false` and asserting that the job is in the database is even better.